src/coreclr/nativeaot/BuildIntegration/WindowsAPIs.txt

```diff
@@ -1394,7 +1394,6 @@ kernel32!SetEndOfFile
 kernel32!SetEnvironmentStringsW
 kernel32!SetEnvironmentVariableA
 kernel32!SetEnvironmentVariableW
-kernel32!SetErrorMode
```

Comment:
This change in this file should be reverted. This list applies to 3rd party libraries too.